### ISSUE TITLE
HttpListenerException error

### ISSUE THREAD

**Issue by DavidIQ** [AntonyCorbett/OnlyT#384]
**Describe the bug**
When enabling the web clock with remote apps enabled the app is crashing (or vice-versa).

**To Reproduce**
1. Go to settings and enable remote apps
2. Attempt to enable web clock
3. Application stops responding and exits

Note that if I disable remote apps I can enable web clock, but then the app crashes if I try to enable remote apps. I was able to enable both by modifying `options.json` and setting both options to `true` before starting the app.

**Expected behavior**
Should be able to enable both.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - Windows version: Windows Server 2019 Standard
 - IIS is also running on same server

Log file attached.

[log-20200324.txt](https://github.com/AntonyCorbett/OnlyT/files/4377604/log-20200324.txt)

**Comment by DavidIQ** [comment]
I realize this isn't going to be a standard setup, but because of the current global situation and with everyone using Zoom, I've set up OnlyT on this remote server so that the sound servant can control remotely and so the brothers giving parts are able to see the clock.

**Comment by bobdallama** [comment]
@DavidIQ could you kindly explain what service or app you are using for this? This would be a tremendous addition to our meetings

**Comment by scottys94** [comment]
#416 
I found that you need to enter a remote Access Code first. Then close/re-open OnlyT. After that, you can enable Remote Apps. If you try to enable Remote Apps without having an access code entered, OnlyT will crash.

**Comment by DavidIQ** [comment]
> @DavidIQ could you kindly explain what service or app you are using for this? This would be a tremendous addition to our meetings

It's a Windows Server that I own so it has its own FQDN (and of course internet IP address) which I remoted into and have set things up for OnlyT. There's a lot more details and steps I had to go through, but the general answer to your question is basically that there is no actual service I'm using. A Windows desktop would work as well for this. Neither option is for the faint at heart.
